flink-table/flink-sql-parser-hive/src/test/java/org/apache/flink/sql/parser/hive/FlinkHiveSqlParserImplTest.java

```diff
@@ -46,6 +46,38 @@ void testTableHintsInInsert() {}
     @Test
     void testDescribeSchema() {}
 
+    /**
+     * Here we override the super method to avoid test error from `ARRAY_AGG` supported in original
```

Comment:
nit: simplify the comments to only an inline comment `// ARRAY_AGG` the explanation in the inline comment above should clarify the rest to avoid code duplication.